src/borg/archiver/diff_cmd.py

```diff
@@ -64,25 +83,61 @@ def build_parser_diff(self, subparsers, common_parser, mid_common_parser):
         from ._common import process_epilog
         from ._common import define_exclusion_group
 
-        diff_epilog = process_epilog(
-            """
-            This command finds differences (file contents, user/group/mode) between archives.
+        diff_epilog = (
+            process_epilog(
+                """
+        This command finds differences (file contents, user/group/mode) between archives.
+
+        A repository location and an archive name must be specified for ARCHIVE1.
+        ARCHIVE2 is just another archive name in same repository (no repository location
+        allowed).
```

Comment:
Replace 89..93 with:

This command finds differences (file contents, metadata) between ARCHIVE1 and ARCHIVE2.

(this was already outdated before your PR)